x/feegrant/module/module.go

```diff
@@ -160,6 +162,22 @@ func (am AppModule) EndBlock(ctx context.Context) error {
 // TxValidator implements appmodulev2.HasTxValidator.
 // It replaces auth ante handlers for server/v2
 func (am AppModule) TxValidator(ctx context.Context, tx transaction.Tx) error {
-	// TODO in follow-up
+	validators := []interface {
+		ValidateTx(ctx context.Context, tx sdk.Tx) error
+	}{
+		ante.NewDeductFeeDecorator(am.accountKeeper, am.bankKeeper, am.keeper, nil),
```

Comment:
this forces an import of auth. can we just put it in auth and call it a day?

Reply:
I can make these changes once after julien's #19949 is finished, as that pr handles most of the antehandler stuff for auth module.